check_code_issue.go

```diff
@@ -0,0 +1,28 @@
+package opslevel
+
+type CodeIssueCheckFragment struct {
+	Campaign string `graphql:"campaign"` // The campaign the check belongs to.
+	Url      string `graphql:"url"`      // The url to the check.
```

Comment:
Same with this field - this is likely on the base check type not specific to the CodeIssuesCheck type.

Reply:
You're right on. `Campaign` and `Url` are both part of the base check type